packages/rsyslog/files/12_rsyslog

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
Why this file has been moved from 20 to 12?

Reply:
From testing and inspection this level (same as original logd) catches the most complete log and we decided this was enough.

Reply:
Change to this file is not related to the boot order. Please revert this change.

Change on init script looks good.